lib/client/beans.html

```diff
@@ -13,8 +13,15 @@
     <script src="js/state/Store.js"></script>
     <script>
         function renderView() {
+            var beans = new BeansView({
+                store: new Store({
```

Comment:
Need to separate that out into it's own variable so that other utilities (apart from UI) can access the store